eZ/Bundle/EzPublishIOBundle/Tests/EventListener/StreamFileListenerTest.php

```diff
@@ -115,9 +120,14 @@ public function testRespondsToIoRequest()
         $this->eventListener->onKernelRequest($event);
 
         self::assertTrue($event->hasResponse());
+        $expectedResponse = new BinaryStreamResponse($binaryFile, $this->ioServiceMock);
+        $response = $event->getResponse();
+        // since symfony/symfony v3.2.7 Response sets Date header if not explicitly set
+        // @see https://github.com/symfony/symfony/commit/e3d90db74773406fb8fdf07f36cb8ced4d187f62
+        $expectedResponse->setDate($response->getDate());
```

Comment:
Note: issue with this has been fixed in symfony/symfony#22304, which hasn't been tagged yet.
However, I would say, we still need this workaround, because there's no guarantee that two `DateTime` objects store the same date/time.